### ISSUE TITLE
[Bug] TXT records with DMARC content rejected by atomiadnsclient

### ISSUE THREAD

**Issue by klaver** [atomia/atomiadns#21]
Hi Atomia,

We love your DNS API, it's working wonderful!

Recently we tried to enable DMARC support for our domains, but the required dataformat does not seem to be accepted by the atomiadnsclient script.
Could you please fix this?

Tried to add the following record:
_dmarc IN 3600 TXT "v=DMARC1; p=none; rua=mailto:postmaster@example.net;"

Returned error message:
"struct parameter needs to have format 'key1 = value2; key2 = value2' at /usr/bin/atomiadnsclient line 110."

**Comment by jimmybergman** [comment]
Yes, this is a known issue, the serialization format used for arguments with atomiadnsclient has no quoting - we are fixing it in the near future.

However as a workaround you can add it directly through the JSON API with curl instead, like:

curl -i -X POST -d '[ "somezone.com", [ { "ttl": 3600, "label": "_dmarc", "class": "IN", "id": -1, "type": "TXT", "rdata": "v=DMARC1; p=none; rua=mailto:postmaster@example.net;" } ] ]' http://localhost/pretty/atomiadns.json/AddDnsRecords; echo

Hope it helps.
